.github/workflows/ci.yaml

```diff
@@ -29,6 +29,8 @@ env:
 jobs:
   python_sdist:
     runs-on: ubuntu-22.04
+    outputs:
+      artifact_name: ${{ steps.build_sdist.outputs.artifact_name }}
```

Comment:
*(this was another missing bit that prevented the wheel jobs from referencing the tarball name)*